cabal.haskell-ci

```diff
@@ -2,6 +2,6 @@ branches: master ci*
 
 -- unconstrained: False
 
-constraint-set bytestring-0.11
-  ghc: == 8.10.7
-  constraints: bytestring >=0.11 && <0.12
+constraint-set bytestring-0.12
+  ghc: >= 9.6
+  constraints: bytestring >=0.12
```

Comment:
This `constraint-set` checks that `base64-bytestring` builds with `bytestring-0.12` on CI.
However, it seems that Haskell CI does not run the tests again for a `constraint-set` build.
I ran the tests locally.